.github/workflows/ci.yml

```diff
@@ -87,3 +87,8 @@ jobs:
         if [[ "$ERROR_COUNT" -gt "0" ]]; then
           exit 1
         fi
+    - name: Test Custom Cloud Shell Image Build
```

Comment:
Short tests like this should go near the top so we can fail fast if anything goes wrong. This step should be right after the license check